internal/provider/client/mte_config_dto.go

```diff
@@ -18,6 +18,7 @@ type RoutesDto struct {
 	CacheKey           *CacheKeyDto   `json:"cacheKey,omitempty"`
 	AppendPathPrefix   string         `json:"appendPathPrefix,omitempty"`
 	ShieldLocation     ShieldLocation `json:"shieldLocation,omitempty"`
+	CacheMaxAge        string         `json:"cacheMaxAge,omitempty"`
```

Comment:
Is the API okay that you send cacheMaxAge when it does nothing with it at the moment?

Reply:
The main API was spun up with this for testing and didn't have any issues handling the tests. I assume that means its ok. Any thoughts?

Reply:
Yep all good